clang/test/AST/Interp/records.c

```diff
@@ -0,0 +1,114 @@
+// UNSUPPORTED: asserts
+// REQUIRES: asserts
+// ^ this attempts to say "don't actually run this test", because it's broken
+//
+// The point of this test is to demonstrate something that ExprConstant accepts,
+// but Interp rejects. I had hoped to express that as the same file with two
```

Comment:
Please stop calling it "Interp", it just sounds silly.

Reply:
😅 sorry, I got that from the abbreviation in the commit messages. 

What do you call it, when you need to distinguish it from the recursive-descent implementation?

Reply:
"The new constant interpreter" or a variation thereof.